pkgs/os-specific/linux/firmware/fwupd/install-fwupdplugin-to-out.patch

```diff
@@ -1,37 +1,104 @@
+-  install_dir: libdir_pkg,
++  install_dir: daemonlibdir,
+   install: true
  )
  
- fwupdplugin_pkgg = import('pkgconfig')
-@@ -280,7 +281,8 @@ if introspection.allowed()
-       girtargets,
-       fwupd_gir[0],
-     ],
--    install: true
-+    install: true,
-+    install_dir_typelib: bindir / '..' / 'lib' / 'girepository-1.0',
-   )
- 
-   # Verify the map file is correct -- note we can't actually use the generated
-diff --git a/meson.build b/meson.build
-index b91dd037..f97b4c26 100644
---- a/meson.build
-+++ b/meson.build
-@@ -504,7 +504,7 @@ if build_standalone
- if host_machine.system() == 'windows'
-   plugin_dir = 'fwupd-plugins-@0@'.format(libfwupdplugin_lt_current)
- else
--  plugin_dir = join_paths(libdir, 'fwupd-plugins-@0@'.format(libfwupdplugin_lt_current))
-+  plugin_dir = join_paths(bindir, '..', 'lib', 'fwupd-plugins-@0@'.format(libfwupdplugin_lt_current))
- endif
- conf.set_quoted('FWUPD_PLUGINDIR', plugin_dir)
+diff --git a/plugins/dfu/meson.build b/plugins/dfu/meson.build
+index 1d51df061..c9362a5de 100644
+--- a/plugins/dfu/meson.build
++++ b/plugins/dfu/meson.build
+@@ -37,7 +37,7 @@ fu_dfu_tool = executable(
+   ],
+   c_args: cargs,
+   install: true,
+-  install_rpath: libdir_pkg,
++  install_rpath: daemonlibdir,
+   install_dir: bindir
+ )
  endif
+diff --git a/plugins/modem-manager/meson.build b/plugins/modem-manager/meson.build
+index 325c55e05..290dbdb72 100644
+--- a/plugins/modem-manager/meson.build
++++ b/plugins/modem-manager/meson.build
+@@ -26,7 +26,7 @@ shared_module('fu_plugin_modem_manager',
+   include_directories: plugin_incdirs,
+   install: true,
+-  install_rpath: libdir_pkg,
+-  install_dir: libdir_pkg,
++  install_rpath: daemonlibdir,
++  install_dir: daemonlibdir,
+   c_args: cargs,
+   link_with: plugin_libs,
+   dependencies: [
+diff --git a/src/meson.build b/src/meson.build
+index ebb262224..19e333972 100644
+--- a/src/meson.build
++++ b/src/meson.build
+@@ -87,7 +87,7 @@ fwupdutil = library(
+   ],
+   install: true,
+-  install_rpath: libdir_pkg,
+-  install_dir: libdir_pkg,
++  install_rpath: daemonlibdir,
++  install_dir: daemonlibdir,
+   include_directories: [
+     root_incdir,
+     fwupd_incdir,
+@@ -128,7 +128,7 @@ fwupdmgr = executable(
+     fwupdutil,
+   ],
+   install: true,
+-  install_rpath: libdir_pkg,
++  install_rpath: daemonlibdir,
+   install_dir: bindir
+ )
+
+@@ -199,7 +199,7 @@ fwupdengine = library(
+   sources: fwupd_engine_src,
+   install: true,
+-  install_rpath: libdir_pkg,
+-  install_dir: libdir_pkg,
++  install_rpath: daemonlibdir,
++  install_dir: daemonlibdir,
+   include_directories: plugin_incdirs,
+   dependencies: [
+     daemon_dep,
+diff --git a/src/meson.build b/src/meson.build
+index b9f6736f2..e9af419af 100644
+--- a/src/meson.build
++++ b/src/meson.build
+@@ -240,7 +240,7 @@ fwupdtool = executable(
+     plugin_libs,
+   ],
+   install: true,
+-  install_rpath: libdir_pkg,
++  install_rpath: daemonlibdir,
+   install_dir: bindir
+ )
```

Comment:
Looks like something ate a line of the patch, leading to 

```
error: corrupt patch at line 105
```

when using `git apply`:

```suggestion
 )
 
```